recipes/protobuf/all/conanfile.py

```diff
@@ -41,10 +42,16 @@ def source(self):
     def configure(self):
         if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
             del self.options.fPIC
```

Comment:
```suggestion

    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
        if self.options.shared:
            del self.options.fPIC
```

Or might be fixed in another PR